shard.yml

```diff
@@ -11,13 +11,13 @@ dependencies:
     version: 0.2.1
   baked_file_system:
     github: schovi/baked_file_system
-    version: 0.9.7
+    version: 0.9.8
```

Comment:
It might be worth considering usage of `~>` operator instead of fixed versions.

Reply:
yeah probably we should do that :slightly_smiling_face: